email-common/src/main/java/io/commercetools/sunrise/email/MessageEditor.java

```diff
@@ -0,0 +1,21 @@
+package io.commercetools.sunrise.email;
+
+import javax.annotation.Nonnull;
+import javax.mail.MessagingException;
+import javax.mail.internet.MimeMessage;
+
+/**
+ * A functional interface for modifying a message without needing to handle checked exceptions like the
+ * {@link MessagingException}s thrown by {@link MimeMessage}.
+ */
+@FunctionalInterface
+public interface MessageEditor {
```

Comment:
@lauraluiz do we here also need test code inclusion like in the JVM SDK or just write examples into the Javadoc?